pex/scie/configure-binding.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Pex project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import print_function
+
+import os
+import sys
+
+
+def write_bindings(
+    env_file,  # type: str
+    installed_pex_dir,  # type: str
+):
+    # type: (...) -> None
+    with open(env_file, "a") as fp:
+        print("PYTHON=" + sys.executable, file=fp)
+        print("PEX=" + os.path.realpath(os.path.join(installed_pex_dir, "__main__.py")), file=fp)
+
+
+if __name__ == "__main__":
+    write_bindings(
+        env_file=os.environ["SCIE_BINDING_ENV"],
+        installed_pex_dir=(
+            # The zipapp case:
+            os.environ["_PEX_SCIE_INSTALLED_PEX_DIR"]
```

Comment:
Should this also be `os.environ.get`? Or is `_PEX_SCIE_INSTALLED_PEX_DIR` guaranteed to exist?

Reply:
It's always set right here in the lift manifest: https://github.com/pex-tool/pex/pull/2466/files/82c629dffaa2aeb2e9a0f9210fcc7a46aeb1c43f#diff-3ed834e0e145c7460bbbb64066efe81446a388f9afce7073d5d77eac77626c79R107

Originally I placed the code inline in the binding command args like so: `..., "-c", dedent("""...""")]`. That made the availability of `_PEX_SCIE_INSTALLED_PEX_DIR` obvious since it was set in `env` a few lines above.